autoscheduler/frontend/src/redux/actions/courseCards.ts

```diff
@@ -55,6 +56,7 @@ export function parseMeetings(arr: any[]): Meeting[] {
       currentEnrollment: Number(sectionData.current_enrollment),
       maxEnrollment: Number(sectionData.max_enrollment),
       instructor: new Instructor({ name: sectionData.instructor_name }),
+      grades: sectionData.grades == null ? null : new Grades({ ...sectionData.grades }),
```

Comment:
I'm pretty sure you don't have to use spreading here, since the constructor just uses `Object.assign` with the parameter passed in

Reply:
Squashed into `Added grades to section type`